Orphanage/orphanage.js

```diff
@@ -0,0 +1,306 @@
+//START SCRIPT
```

Comment:
The code looks good. File an issue in the tracker to refactor the code and assign it to yourself. We can pair on the task in Austin during All Hands.